nucypher/policy/conditions/json/rpc.py

```diff
@@ -0,0 +1,177 @@
+from abc import ABC
+from typing import Any, Optional, Tuple
+
+from marshmallow import ValidationError, fields, post_load, validate, validates
+from marshmallow.fields import Url
+from typing_extensions import override
+
+from nucypher.policy.conditions.context import is_context_variable
+from nucypher.policy.conditions.exceptions import (
+    JsonRequestException,
+)
+from nucypher.policy.conditions.json.base import (
+    HTTPMethod,
+    JSONPathField,
+    JsonRequestCall,
+)
+from nucypher.policy.conditions.json.utils import process_result_for_condition_eval
+from nucypher.policy.conditions.lingo import (
+    ConditionType,
+    ExecutionCallAccessControlCondition,
+    ReturnValueTest,
+)
+
+
+class BaseJsonRPCCall(JsonRequestCall, ABC):
+    class Schema(JsonRequestCall.Schema):
+        method = fields.Str(required=True)
+        params = fields.Field(required=False, allow_none=True)
+        query = JSONPathField(required=False, allow_none=True)
```

Comment:
If I understand it correctly, queries for `JsonRpcCalls` will be parsed not from the json root, but from the `result` field, right? If so, this should be explicit somewhere.

Reply:
Correct. When you say "explicit", what do you mean exactly? Something like `query_from_result` instead of `query` for the field name or something like that? I was hoping documentation would be sufficient.